pybossa/model/event_listeners.py

```diff
@@ -294,6 +294,15 @@ def on_taskrun_submit(mapper, conn, target):
         push_webhook(project_private, target.task_id, result_id)
 
 
+@event.listens_for(TaskRun, 'after_update')
+def on_taskrun_resubmit(mapper, conn, target):
+    """Reset task exported flag when taskrun is modified."""
+    task = task_repo.get_task(id=target.task_id)
```

Comment:
This line will make a query to task table. How about removing this line? For the line 301 and line 302, you can use target.task_id as the task ID.

Reply:
updated. Thanks!